extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -88,6 +109,9 @@ function SidebarWithRef(
   // `mounted` mounts the component. `opened` plays the animation.
   const [mounted, setMounted] = useState(false);
   const [opened, setOpened] = useState(false);
+  const [side, setSide] = useState(
+    typeof document === 'object' ? calculateSide(sideProp, document) : sideProp
```

Comment:
Hmm. If the actual `sideProp` is specified, we should just respect the value given. So I think we need to reverse this, or do something like: `sideProp || (... calculateSide)`